evm/p2p/peer.py

```diff
@@ -696,7 +636,7 @@ def __init__(self, block_number, block_hash, total_difficulty, genesis_hash):
         self.genesis_hash = genesis_hash
 
 
-if __name__ == "__main__":
+def _test():
```

Comment:
Is this temporary for development?  Is there a plan to convert this to a proper test?  Convenience of some sort?

Reply:
It could become an integration test, similar to the one for the lightchain, but in this branch I'm just moving it inside a function to avoid having all the variables defined here in the module's global namespace